### DISCUSSION TITLE
What is the aim of this repository?

### DISCUSSION THREAD

**Discussion by t-inamura** [HumanSupportRobot/QandA#1] [Q&A] [answered]
This repository does not seem like a source code sharing.
What is the aim of this repository?
Who manages this repository?

**Comment by k-takeshita** [comment, marked as answer]
This repository is for QA for current and future users of Human Support Robot.
http://humansupportrobot.org/

**Reply by t-inamura** [reply]
Thanks. I understood.

**Comment by hsr-admin** [comment]
For technical questions, please follow the STEP below to attempt to solve them.
[STEP 1] Investigation by yourself
Please check FAQ on hsr site to solve the problem by yourself.
[STEP2] Ask question to this QA site
Please register it as a new question at Discussions in this github repository.
Everybody can answer it. It is expected to solve problems by mutual cooperation among members.
